app/services/trading_strategies.py

```
@@ -4153,6 +4153,405 @@ def _calculate_portfolio_sortino(self, positions: List[Dict]) -> float:
+            
+            # Apply adjustment if provided
+            adjusted_distance = liquidation_distance * (1 + adjustment)
+            
+            if side.lower() == "long":
+                liquidation_price = entry_price * (1 - adjusted_distance)
+            else:  # short
+                liquidation_price = entry_price * (1 + adjusted_distance)
+                
+            return max(0, liquidation_price)
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)